telegram/ext/messagehandler.py

```diff
@@ -66,9 +68,13 @@ def __init__(self,
                  pass_user_data=False,
                  pass_chat_data=False,
                  message_updates=True,
-                 channel_post_updates=True):
-        if not message_updates and not channel_post_updates:
-            raise ValueError('Both message_updates & channel_post_updates are False')
+                 channel_post_updates=True,
+                 edited_updates=False):
+        if not message_updates and not channel_post_updates and not edited_updates:
+            raise ValueError(
+                'message_updates, channel_post_updates and edited_updates are all False')
+        if allow_edited:
+            warnings.warn('allow_edited is getting deprecated, please use edited_updates instead')
```

Comment:
I'm not renaming an argument, I'm adding behavior and warning the old behavior is going to be deprecated.
What you suggest would be valid if `allow_edited` would reflect the change but it doesn't.
 If I do it this way, it will implement the new functionality and still work with the old way. If you want to remove the deprecated way in the future you need to remove three lines in the code instead of removing thee and modifying two.